packages/chat-component-bindings/review/beta/chat-component-bindings.api.md

```diff
@@ -49,7 +49,7 @@ export type ChatHandlers = {
     onLoadPreviousChatMessages: (messagesToLoad: number) => Promise<boolean>;
     onUpdateMessage: (messageId: string, content: string, options?: {
         metadata?: Record<string, string>;
-        attachmentMetadata?: AttachmentMetadata[];
```

Comment:
how are we going to update inline images in chat message when we released inline image upload feature later?

Reply:
> how are we going to update inline images in chat message when we released inline image upload feature later?

Hi @jpeng-ms , that's a good question. We can always introduce the API back in when we allow for inline images to be sent from ACS -> Teams. However, we want to having a safer / smaller API surface which means we don't introduce APIs that we don't need at this moment.

Reply:
i feel like changing something back and forth would not be any safer at all, and on the contrary, it's error prone and problematic